awx/main/tasks.py

```diff
@@ -3,8 +3,7 @@
 
```

Comment:
For multiple types of jobs, we call `add_awx_venv`, notably, inventory updates and system jobs.

Since the Ansible venv this is supporting is python 2.7... might that fail some weird scenarios?

Maybe adding the AWX venv to these things _shouldn't_ happen.